### ISSUE TITLE
Not blocking 

### ISSUE THREAD

**Issue by Sczar** [GoldenKappa/notSABS#1]
After tthe latest update CRB9 for the S8+ the app no longer block.

**Comment by hassanila97** [comment]
It works on new updates also. What you need to do is update the lists manually and reactivate SABS.

Here are the working lists:
https://raw.githubusercontent.com/GoldenKappa/notSABS/master/standard-package.txt

And  

https://raw.githubusercontent.com/GoldenKappa/notSABS/master/standard-package-mmotti.txt

**Comment by Sczar** [comment]
That did it Hassan. Thank you.

**Comment by cammybird** [comment]
Hassan, where do you update the lists.   A little help would be appreciated, I also have SABS installed and working.  It doesn't block anything.  I'm using Android Studio.   THANKS.
.

**Comment by cammybird** [comment]
That's it......thanks.

**Comment by deniz87** [comment]
My SABS stopped blocking ads for the last few days. I see ads everywhere now and "Recent Activity" page is empty. I still have the mmotti's package enabled and my knox key is still valid on samsung's page. Package Disabler is still working though. Is this a common issue and what can I do to make it block the ads again?

**Comment by HiHat** [comment]
More recent MMotti's list here:
https://github.com/mmotti/mmotti-host-file/raw/master/hosts

**Comment by Stan-Tastic** [comment]
Perhaps I'm doing something wrong, but even after doing "Update selected" and notSABS stop/start, the number of blocked domains is "0", as indicated by number under name of blocklist and "blocked unique domains". Also I can see that I'm still getting ads everywhere.

What did I miss?

**Comment by deniz87** [comment]
@Stan-Tastic 
I had the same issue and I found a fix for my own case. I explained this on Zozzle website too. 

Try this;
Hit the "TURN OFF" button and quickly TURN ON (turning on was taking 1-3 sec, which is bad and it was the issue) and consecutively keep turning on and off until turning on takes 20-30 secs instead of 1-3 secs. Then it ACTUALLY turns on and starts working. This is how I fixed it.

I think going into the strongest power savings mode on my s9+ broke my SABS in my case.

**Comment by Stan-Tastic** [comment]
@deniz87 not sure if we're on the same page - in "Managed blocked domains/Subscribe to providers" it shows 0 entries in both Standard and mmotti's package, despite clicking "update selected" multiple times.

Nevertheless, for past 30 minutes I've been disabling and enabling SABS, and it always takes 2 seconds to start.

SGS7, Oreo.

**Comment by deniz87** [comment]
Yup, even updating the lists manually didn't fix it for me. Only the way I told you did. Try it again after restarting the phone. I did it very fast. It took me maybe only 20 tries until the stage it takes 20-30 secs to enable successfully.
